jdbc-core/build.gradle.kts

```diff
@@ -1,5 +1,6 @@
 plugins {
     id("java-conventions")
+    id("java-test-fixtures")
```

Comment:
This should be made unnecessary by #72

Reply:
makes sense, let's see which one lands first

Reply:
Yeah `HyperResultSourceTest.scala` would need to implement a small shim, this is because #72 tries to avoid adding jdbc-core concept to what should just be the hyperd management code